### ISSUE TITLE
Custom Link: Probleme mit Sonderzeichen

### ISSUE THREAD

**Issue by Hirbod** [FriendsOfREDAXO/mform#217]
<img width="876" alt="Bildschirmfoto 2021-03-25 um 17 15 10" src="https://user-images.githubusercontent.com/504909/112506094-b61a1580-8d8d-11eb-9be8-817997b17f68.png">

**Comment by skerbis** [comment]
Welches Sonderzeichen? was soll o? sein?
<img width="575" alt="Bildschirmfoto 2021-03-26 um 17 59 24" src="https://user-images.githubusercontent.com/791247/112666989-09a76480-8e5d-11eb-8e9d-aeb0a7b4a091.png">

**Comment by Hirbod** [comment]
Ist bei mir ein Ö

**Comment by skerbis** [comment]
<img width="546" alt="Bildschirmfoto 2021-03-26 um 18 05 22" src="https://user-images.githubusercontent.com/791247/112667690-de714500-8e5d-11eb-8c7b-62df69f1035f.png">
hmm 🙄

**Comment by Hirbod** [comment]
PHP 8 vllt noch erwähnenswert

**Comment by skerbis** [comment]
Kann ich gerade auf meinem System live nicht testen. Alle Locales installiert?

**Comment by Hirbod** [comment]
Ja, sind sie. Lass das Ticket mal einfach offen, ich debugge das später selbst mal, gibt vermutlich irgendeinen Edge Case, der das Problem erklären wird.
